test/karma.conf.js

```diff
@@ -121,25 +127,21 @@ module.exports = function(config) {
       'karma-firefox-launcher',
       'karma-chrome-launcher',
       'karma-phantomjs-launcher',
+      'karma-nightmare',
       'karma-ng-html2js-preprocessor',
       'karma-jasmine',
-      'karma-coverage'
+      'karma-coverage',
+      'karma-jasmine-diff-reporter'
     ],
 
     // Continuous Integration mode
     // if true, it capture browsers, run tests and exit
-    singleRun: false,
-
-    colors: true,
+     singleRun: false,
 
     // level of logging
     // possible values: LOG_DISABLE || LOG_ERROR || LOG_WARN || LOG_INFO || LOG_DEBUG
-    logLevel: config.LOG_DEBUG,
+    logLevel: config.LOG_ERROR,
```

Comment:
Any specific reason to change this from debug?

Reply:
It was just noisy, was trying to cut down on stuff to swim through. Let me roll that back.